apps/webClient/src/components/NavButton/index.tsx

```diff
@@ -7,37 +7,56 @@ import { Theme } from "@material-ui/core/styles/createMuiTheme";
 interface Props {
   route: string;
   text: string;
+  vertical: boolean;
 }
 
 const useStyles = makeStyles((theme: Theme) => ({
-  ourWork: {
-    backgroundColor: "white",
+  verticalBtn: {
+    backgroundColor: "transparent",
     borderRadius: 0,
+    border: "solid 3px white",
     width: 60,
     height: 200,
   },
-  aboutUs: {
-    backgroundColor: "transparent",
+  horizontalBtn: {
+    backgroundColor: "white",
     borderRadius: 0,
-    border: "solid 4px white",
-    width: 60,
-    height: 200,
+    width: 200,
+    height: 60,
+    "&:hover": {
+      backgroundColor: "white",
+    },
+  },
+  verticalLabelRight: {
+    writingMode: "vertical-rl",
+    textOrientation: "sideways",
+    fontSize: 20,
+    textTransform: "uppercase",
   },
-  buttonLabel: {
+  verticalLabelLeft: {
     writingMode: "vertical-rl",
     textOrientation: "sideways",
     fontSize: 20,
     textTransform: "uppercase",
+    transform: "rotate(-180deg)",
   },
```

Comment:
in this case, because verticalLabelLeft and right are so close, I think it makes sense to have one verticalLabel class, and a separate `verticalLabelLeft` class which you can also use for left